nextmove/src/main/java/no/difi/meldingsutveksling/nextmove/logging/ConversationResourceMarkers.java

```diff
@@ -14,7 +14,21 @@ public static LogstashMarker markerFrom(ConversationResource resource) {
         final LogstashMarker conversationIdMarker = MarkerFactory.conversationIdMarker(resource.getConversationId());
         final LogstashMarker senderIdMarker = MarkerFactory.senderMarker(resource.getSenderId());
         final LogstashMarker receiverMarker = MarkerFactory.receiverMarker(resource.getReceiverId());
-        final LogstashMarker messagetypeIdMarker = MarkerFactory.messageTypeMarker(resource.getServiceIdentifier().toString());
+        final LogstashMarker messagetypeIdMarker = MarkerFactory.messageTypeMarker(serviceIdentifierLoggingOverride(resource));
         return conversationIdMarker.and(senderIdMarker).and(receiverMarker).and(messagetypeIdMarker);
     }
+
+    /**
+     * Override for service identifier logging, in case of meeting message. Should be removed with the next version of NextMove.
+     * @param cr ConversationResource
+     * @return serviceIdentifier used for logging
+     */
+    public static String serviceIdentifierLoggingOverride(ConversationResource cr) {
+        if (cr.getCustomProperties() != null &&
+                cr.getCustomProperties().containsKey("meeting") &&
+                cr.getCustomProperties().get("meeting").equals("true")) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Move the "true" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1132)